### PR TITLE
cloud: add security settings

### PR DESCRIPTION
**Pull request by qiancai**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by xuechunL — APPROVED** [review]
LGTM

**Review by zhoubasten — APPROVED** [review]
LGTM

File: tidb-cloud/configure-security-settings.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+2. Click **Security Quick Start** in the pane. The **Security Quick Start** dialog is displayed.
+3. In the dialog, configure the root password and allowed IP addresses.
+
+    To allow your cluster to be accessible by any IP addresses, click **Allow Access From Anywhere**.
+
+4. Click **Apply**.
+
+> **Tip:**
+>
+> If you are viewing the overview page of your cluster, you can click the **Quick Start** tab on the page and configure these settings too.
```
Comment (TomShawn) [inline tidb-cloud/configure-security-settings.md:22]:
```suggestion
> If you are viewing the overview page of your cluster, you can click the **Quick Start** tab on the page and configure these settings, too.
```

File: tidb-cloud/configure-security-settings.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+---
+title: Configure Cluster Security Settings
+summary: Learn how to configure the root password and allowed IP addresses to connect to your cluster.
+---
+
+# Configure Cluster Security Settings
+
+When you create a cluster, the TiDB Cloud console will automatically guide you to configure the root password and allowed IP addresses to connect to your cluster.
+
+If you do not configure the root password or allowed IP addresses at that time, or if you want to modify the settings, take the following steps:
```
Comment (TomShawn) [inline tidb-cloud/configure-security-settings.md:10]:
```suggestion
If you do not configure the root password or allowed IP addresses at that time, or if you want to modify the security settings, take the following steps:
```

**Review by TomShawn — APPROVED** [review]
rest LGTM

**Comment by qiancai** [comment]
@TomShawn Thanks for the comments. I've accepted them. PTAL.

**Comment by TomShawn** [comment]
LGTM

**Pull request merged**